debezium-core/src/test/java/io/debezium/transforms/partitions/PartitionRoutingTest.java

```diff
@@ -134,7 +134,7 @@ public void onlyFieldThatExistForCurrentEventWillBeUsedForPartitionComputation()
 
         SourceRecord transformed = partitionRoutingTransformation.apply(eventRecord);
 
-        assertThat(transformed.kafkaPartition()).isOne();
+        assertThat(transformed.kafkaPartition()).isZero();
```

Comment:
@vjuranek Could you please change the test data in a way that has is non-zero? Zero could be default value so let's make the test more robust.